sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -331,8 +331,12 @@ export class EventHubConsumerClient {
   /**
    * Subscribe to all messages from a subset of partitions.
    *
-   * Use this overload if you want to read from a specific set of partitions and not coordinate with
+   * Use this overload if you want to read from a specific partition and not coordinate with
    * other subscribers.
+   * 
+   * Note: Reading directly from the same partition with the same CheckpointStore 
+   * and different EventHubConsumerClients will result in OwnershipLost errors as the
+   * subscribers compete to own the partition.
```

Comment:
When receiving from a single partition, the checkpoint store should not be used by the library for ownership management at all...

Reply:
Why not? It works just fine with the way we split out the load balancer.

Reply:
Updatd.